flexbox/nasa.html

```diff
@@ -0,0 +1,42 @@
+<DOCTYPE! html>
+   <html lang="en">
+    <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <meta http-equiv="X-UA-Compatible" content="ie=edge">
+      <title>Nasa</title>
+      <link rel="stylesheet" href="nasa.css">
```

Comment:
make sure when add file paths you include the `./`
`<link rel="stylesheet" href="./nasa.css">`